### PR TITLE
🚢 Ready to be shipped Demo Unit 1: Find your Game Idea and write your GDD

### PR DESCRIPTION
**Pull request by simoninithomas**
🟢 READY TO BE REVIEWED

### CODE REVIEW

File: units/en/demo1/game-design-document.mdx [hunk 1]
```diff
@@ -0,0 +1,223 @@
+# Step 2: Let's write the Game Design Document ✍️
+
+## What is a Game Design Document (GDD)?
+
+A Game Design Document (GDD) is an essential tool in the game industry: it'sit's the **blueprint of a video game project**.
```
Comment (Ivan-267) [inline units/en/demo1/game-design-document.mdx:5]:
```suggestion
A Game Design Document (GDD) is an essential tool in the game industry: it's the **blueprint of a video game project**.
```

File: units/en/demo1/game-design-document.mdx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+The GDD **ensures a clear definition and understanding of the game's vision for all teammates** and facilitates the development process.
+
+In the case of our demo, **we want to write a concise one-page document**.
+
+## The One Page Game Design Document Template
+
+This is our One Page GDD template, you can naturally add or remove elements based on your needs.
+
+You can download the Google Doc version 👉 [here](https://docs.google.com/document/d/15xThlHwa8cA8AMFW6t29TeV2DwoNgjnBPOTc0bnaJ8E/edit?usp=sharing)
```
[the remark this answers is not in the document]
Comment (simoninithomas) [reply units/en/demo1/game-design-document.mdx:23]: Thanks 🤗

[the remark this answers is not in the document]
Comment (simoninithomas) [reply units/en/demo1/game-design-document.mdx:23]: Oh yes I see why the typos. It's because Grammarly didn't replaced the words with ' but added them.

**Review by simoninithomas — COMMENTED** [review]
Thanks for your help on updating the course 🤗

**Comment by Ivan-267** [comment]
> Thanks for your help on updating the course 🤗

You're welcome! The course idea is interesting, best of luck with the course.

**Pull request merged**